shared/src/ledger/ibc/handler.rs

```diff
@@ -858,6 +867,25 @@ pub trait IbcActions {
         Ok(())
     }
 
+    /// Set the timestamp and the height for the client update
+    fn set_client_update_time(&self, client_id: &ClientId) -> Result<()> {
+        let timestamp = Timestamp::now();
```

Comment:
I think will probably be different timestamp on each node. Won't it break consensus in the storage?

Reply:
The timestamp will be checked if a client state is valid on this node.

Reply:
Thank you for the suggestion! In ibc-rs, the time is acquired from Tendermint header.
I've fixed it.